### PR TITLE
Update VarTracker.java

### PR DESCRIPTION
**Pull request by Ayush-Thakur-geek**
FOR issue #380

### CODE REVIEW

**Comment by cyrille-artho** [comment]
Thanks, your patch looks good. Could you please try to add a unit test as well so that we have a failing test (from your example) that shows that the patch is needed?

File: src/classes/modules/java.logging/java/util/logging/FileHandler.java
Comment (cyrille-artho) [inline src/classes/modules/java.logging/java/util/logging/FileHandler.java:1]: This change is not necessary; the module name is not part of the package name (but it is reflected in the file name).

**Review by cyrille-artho — CHANGES_REQUESTED** [review]
The new change is unrelated and not needed; please revert it.
Instead, please try to add a unit test that fails if your fix to `VarTracker` is not applied yet.

**Comment by abimael-turing** [comment]
@cyrille-artho If you would like, you can assign this issue to me!

**Pull request closed without merging**